### ISSUE TITLE
Android Chrome playlist DOM exception on playlist transition

### ISSUE THREAD

**Issue by blacktrash** [flowplayer/flowplayer#1376]
https://flowplayer.blacktrash.org/fp7/playlist.html
Using 7.2.5 to safely discard https://github.com/flowplayer/flowplayer/issues/1375
Click prev or next or playlist item.
Could be because hls.js instance not destroyed for next clip.

**Comment by blacktrash** [comment]
FYI, with 7.2.5 I am able avoid  with this ugly hack:
```js
api.on('load', function (e, api, video) {
  if (api.conf.playlist && api.conf.playlist.length) {
    var hls = flowplayer.engine('hlsjs-lite').hls;
    if (hls) {
      hls.destroy();
      api.load(video);
    }
  }
});
```

**Comment by nnarhinen** [comment]
Are you still able to reproduce this with the latest dev (7.2.7) assets? Unfortunately you need to clone & make yourself, no automated build at this point.

**Comment by blacktrash** [comment]
Still broken: https://flowplayer.blacktrash.org/fp7/playlist-basic-js.html now uses dev build.

**Comment by blacktrash** [comment]
Actually, this _seems_ to be solved if I remove splash which triggers #1375 .
I obviously cannot test whether it _would_ work in splash setups because it breaks at the first load.

**Comment by nnarhinen** [comment]
What devices have you been using to test? I cannot seem to reproduce this with any browserstack devices.

**Comment by blacktrash** [comment]
See https://github.com/flowplayer/flowplayer/issues/1375#issuecomment-404800435 - any Android Chrome at https://flowplayer.blacktrash.org/fp7/playlist.html

**Comment by nnarhinen** [comment]
Did the fix for #1375 fix this too?

**Comment by blacktrash** [comment]
Yes.
